### ISSUE TITLE
6969,6970.Service port problem

### ISSUE THREAD

**Issue by jiaokele** [paulov-t/SIT.Core#91]
6969, 6970, are there any restrictions on the ports? After changing to other ports, personal profiles cannot be loaded. In some configuration files of "SIT.Aki-Server-Mod", changing the ports 6969 and 6970 allows joining the game, but jumping is not possible. Are there any port restrictions? Can the ports be changed later?

**Comment by SlejmUr** [comment]
Yes, there is no strict restriction, the latest client implemented such a thing you can edit your port rather than a hardcoded 6970 ws port.

**Comment by KoHAlucard88** [comment]
> 6969, 6970, are there any restrictions on the ports? After changing to other ports, personal profiles cannot be loaded. In some configuration files of "SIT.Aki-Server-Mod", changing the ports 6969 and 6970 allows joining the game, but jumping is not possible. Are there any port restrictions? Can the ports be changed later?

Hello, i can't jump to, i can't open doors, IA can't die and idk why :/

**Comment by KoHAlucard88** [comment]
> @KoHAlucard88 You need to supply error logs otherwise nobody can help. There are three logs. Server (you'll see it in the server window). Bepinex (logoutput.log) and Tarkov itself (Logs folder). I guarantee you've missed an installation step.

Ok i make a new install and if i have a problem i create a new post.
